src/PlaywrightSharp.Tests/SelectorsRegisterTests.cs

```diff
@@ -38,6 +38,7 @@ public async Task ShouldWork()
             var page = await context.NewPageAsync();
             await page.SetContentAsync("<div><span></span></div><div></div>");
 
+            /* I'd rather ignore these tests instead of exposing a CreateSelectorForTestAsync function

```

Comment:
Can we remove them entirely? Having stale code in the code base is probably not helpful.